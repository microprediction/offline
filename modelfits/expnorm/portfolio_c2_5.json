{"g1": 0.040383944820019994, "g2": 8.833808685057559, "logK": 0.9622197470180167, "loc": 0.03195469104716461, "logScale": -0.3988849625618007}