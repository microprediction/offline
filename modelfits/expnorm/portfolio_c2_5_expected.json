{"g1": 0.1975371486072879, "g2": 0.19236348868965053, "logK": 0.8278466602629806, "loc": -0.06434635350242904, "logScale": -1.9749177858617424}